jck/jtrunner/JavatestUtil.java

```diff
@@ -534,37 +546,47 @@ else if ( !tests.contains("api/java_awt/interactive") ) {
 			}
 			
 			fileContent += "concurrency " + concurrencyString + ";\n";
-                       
-                        if (!timeoutFactorString.equals("NULL")) {
-				fileContent += "timeoutfactor " + timeoutFactorString + ";\n";
-                        } else {
-				fileContent += "timeoutfactor 4" + ";\n";	// 4 base time limit equal 40 minutes
+			generatedJti.setProperty("concurrency", concurrencyString);
+            if (timeoutFactorString.equals("NULL")) {
+				timeoutFactorString = "4";  // 4 base time limit equal 40 minutes
 			}
+			fileContent += "timeoutfactor " + timeoutFactorString + ";\n";
+			generatedJti.setProperty("timeoutfactor", timeoutFactorString);
```

Comment:
I suspect both "concurrency" and "timeoutfactor" are generic to all tests... so can probably remain in JTB ?